features/account.feature

```diff
@@ -20,8 +20,8 @@ Feature: Account API
 
   Scenario: Get non-existent account
     When an authenticated "get" request is made to "/.ghost/activitypub/account/@nonexistent@fake-external-activitypub"
-    Then the request is rejected with a 500
```

Comment:
😬 Yeah I think we should never have an explicit 500 check here.

I think we should update that stepdef to throw an error if a 5xx error is returned - what do you think?

Reply:
Yeh I don't think it makes sense to expect an internal server error 😅 I'll add a check into the stepdef 👍